packages/components/src/components/code-snippet/_code-snippet.scss

```diff
@@ -333,7 +333,6 @@
     position: absolute;
     top: 0;
     right: 0;
-    z-index: 10;
```

Comment:
@emyarod I removed this as it was causing issues with multiple Code Snippets (#7605) and I'm not seeing any adverse effects, but wanted to make sure I'm not missing a particular use-case

Reply:
I would investigate a bit more since I recall explicitly adding this (https://github.com/carbon-design-system/carbon/pull/7229/commits/6f00446742b007e78a24761d74c4e89dc093d986) to resolve something but it may be fine to remove for now until we spot the regression

Reply:
Noticed an issue with multi-line snippets without the `z-index`, so I just moved the `z-index` to only target that variant 👍 

<img width="1014" alt="Screen Shot 2021-01-21 at 9 56 17 AM" src="https://user-images.githubusercontent.com/11928039/105392251-d8e24f00-5bcf-11eb-87a1-d9ab0ed86e6a.png">